todo/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+
+class Tag(models.Model):
+    name = models.CharField(max_length=100)
+
+    class Meta:
+        ordering = ["name"]
+
+    def __str__(self):
+        return f"{self.name}"
+
+
+class Task(models.Model):
+    content = models.CharField(blank=True, max_length=15)
+    created_at = models.DateTimeField(auto_now_add=True)
```

Comment:
Why content max_length only 15 ? Also I suppose that Task could have name, and content as a TextField